### ISSUE TITLE
in editor editing

### ISSUE THREAD

**Issue by des-des** [dwyl/ampl#14]
This might be better discussed in adoro but came across this: https://zenoamaro.github.io/react-quill/ . . might be worth looking at

**Comment by nelsonic** [comment]
@des-des yes, looks promising! we should ask @zenoamaro if he will accept a Pull Request updating the project from ![build-unknown](https://camo.githubusercontent.com/86ba311c1d23537f55790814e6156be561815fe1/68747470733a2f2f7472617669732d63692e6f72672f7a656e6f616d61726f2f72656163742d7175696c6c2e7376673f6272616e63683d6d6173746572) to something a bit more ... 
![build-passing](https://raw.githubusercontent.com/dwyl/repo-badges/master/highresPNGs/build-passing.png)

**Comment by des-des** [comment]
:+1:
